### PR TITLE
fix: commit message overflows box (#5043)

### PR DESCRIPTION
**Pull request by reginaelyse**
This will fix issue #5043. Added an Expand button that will open a popup with information relating to the commit information if the user wishes to view it in its entirety. This will prevent longer commit messages from overflowing their container. 

info-popup

Checklist:

* [x] Either (a) I've created an [enhancement proposal](https://github.com/argoproj/argo-cd/issues/new/choose) and discussed it with the community, (b) this is a bug fix, or (c) this does not need to be in the release notes.
* [x] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] I've updated both the CLI and UI to expose my feature, or I plan to submit a second PR with them.
* [ ] Does this PR require documentation updates?
* [ ] I've updated documentation as required by this PR.
* [x] Optional. My organization is added to USERS.md.
* [x] I have signed off all my commits as required by [DCO](https://github.com/argoproj/argoproj/tree/master/community#contributing-to-argo)
* [x] My build is green ([troubleshooting builds](https://argoproj.github.io/argo-cd/developer-guide/ci/)).

### CODE REVIEW

File: ui/src/app/applications/components/application-status-panel/application-status-panel.scss [hunk 1]
```diff
@@ -11,7 +11,7 @@
     box-shadow: 1px 2px 3px rgba(0, 0, 0, 0.1);
 
     &__item {
-        height: 75px;
+        max-height: 100px;
```
Comment (rbreeze) [inline ui/src/app/applications/components/application-status-panel/application-status-panel.scss:14]: I think this should still be set at 75px. For shorter errors, setting at 100px leaves an awkward gap: 

Screen Shot 2021-01-14 at 11 02 40 AM

File: ui/src/app/applications/components/utils.tsx [hunk 2]
```diff
@@ -466,7 +466,7 @@ export function getOperationType(application: appModels.Application) {
     return 'Unknown';
 }
 
-const getOperationStateTitle = (app: appModels.Application) => {
+export const getOperationStateTitle = (app: appModels.Application) => {
```
Comment (rbreeze) [inline ui/src/app/applications/components/utils.tsx:469]: Since this is exported now, can you rename it to `GetOperationStateTitle` for convention?

**Review by rbreeze — CHANGES_REQUESTED** [review]
Just a couple small comments, otherwise looks good!

File: ui/src/app/applications/components/utils.tsx (hunk 2 above)
Comment (alexmt) [reply ui/src/app/applications/components/utils.tsx:469]: I don't think this is Javascript convention, only Golang requires a capital case for exported types/functions. 

We've not use capital case convention in Argo CD UI, so I would keep it as is.

File: ui/src/app/applications/components/application-status-panel/application-status-panel.scss (hunk 1 above)
Comment (rbreeze) [inline ui/src/app/applications/components/application-status-panel/application-status-panel.scss:14]:
```suggestion
        max-height: 75px;
```

**Review by rbreeze — APPROVED** [review]

**Pull request merged**